score/compute_score.py

```diff
@@ -34,16 +38,37 @@ def compute_score(config, data, fake_data=None):
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description=__doc__)
-    parser.add_argument("--configuration", required=True,
+    parser.add_argument("--configuration",
         help="frozen benchmark configuration generated by generate_score_config.py")
     parser.add_argument("--benchmark_data_file",
         help="pytest-benchmark json file with current benchmark data")
     parser.add_argument("--benchmark_data_dir",
         help="directory containing multiple .json files for each of which to compute a score")
     parser.add_argument('--hack_data', nargs=2, action='append', help="keyword to match benchmark names, and multiplicative factor to adjust their measurement")
     args = parser.parse_args()
-    with open(args.configuration) as cfg_file:
-        config = yaml.full_load(cfg_file)
+
+    if args.configuration is not None:
```

Comment:
does it make sense to reduce lines of code by putting the config-file generation into the existing code blocks 
if args.benchmark_data_file is not None:
elif args.benchmark_data_dir is not None:
?

You can still have a check at line 53
else:
   if args.benchmark_data_file is None and args.benchmark_data_file is None, raise Exception "invalid arg.."

Reply:
That makes sense. I have updated the code.